apps/web/pages/index.tsx

```diff
@@ -13,7 +13,7 @@ export async function getServerSideProps({ req, res }: GetServerSidePropsContext
     return { redirect: { permanent: false, destination: "/auth/login" } };
   }
 
-  return { redirect: { permanent: false, destination: "/event-types" } };
+  return { redirect: { permanent: false, destination: `/${session.user.homeView}` || "event-types" } };
```

Comment:
Minor: even if empty, the first value would at least be `/`, so the fallback value won't be reached with this approach.

Maybe moves it inside the `${...}` ?

Reply:
yeah you are right